src/main/scala/Day3/Functional1.scala

```diff
@@ -0,0 +1,18 @@
+package Day3
+
+import java.util
+
+object Functional1 extends App{
+
+  val allData = java.util.TimeZone.getAvailableIDs.mkString(", ").split("/").mkString(", ")
+  println(allData)
+
+  val availableIds = java.util.TimeZone.getAvailableIDs.mkString(", ").split("/").tail.mkString(", ").split(", ").grouped(2).flatMap(_.take(1)).toArray.mkString(", ")
```

Comment:
`java.util.TimeZone.getAvailableIDs.mkString(", ").split("/") ` <-- you're repeating this like 3 times pal